src/NuGet/Microsoft.CodeAnalysis.Build.Tasks.nuspec

```diff
@@ -46,9 +46,9 @@
     </contentFiles>
   </metadata>
   <files>
-    <file src="Dlls\MSBuildTask\Microsoft.Build.Tasks.CodeAnalysis.dll" target="lib\netstandard1.3" />
-    <file src="Dlls\MSBuildTask\Microsoft.CSharp.Core.targets" target="contentFiles\any\any" />
-    <file src="Dlls\MSBuildTask\Microsoft.VisualBasic.Core.targets" target="contentFiles\any\any" />
+    <file src="Dlls/MSBuildTask/Microsoft.Build.Tasks.CodeAnalysis.dll" target="lib/netstandard1.3" />
```

Comment:
Please convert any others as well that you know of.  Really I should just move them all in here though.